src/main/scala/io/iohk/ethereum/db/dataSource/DataSource.scala

```diff
@@ -0,0 +1,26 @@
+package io.iohk.ethereum.db.dataSource
+
+trait DataSource {
```

Comment:
@whysoserious we have found at least two data sources (iodb and levelDb) that allow to do batch modifications. Also, we found that we weren't using single items insertions. That being said, if we think it's needed in the future we can easily add it then, how does it sound?